### PR TITLE
Major change here, switching out with the new Launcher

### PR DESCRIPTION
**Pull request by stjohnjohnson**
[Feature](https://github.com/screwdriver-cd/screwdriver/issues/77)

This uses the new launcher and has a minimal image (16.14mb)

### CODE REVIEW

**Comment by Filbird** [comment]
👍

**Comment by jer** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -1,37 +1,28 @@
...
 ```
 
-Note:
- - `git_branch` defaults to `master`
- - `job_name` defaults to `main`
+## Caveats
+
+The container must have a proper init (pid 1) script that can handle reaping zombies.  That's why we use [tini].
+
+Alpine linux does not include glibc, but you can [use musl as it's compatible][glibc].
```
Comment (nkatzman) [inline README.md]: doesn't have git installed.... yet...

File: README.md [hunk 2]
```diff
@@ -1,37 +1,30 @@
...
- - Execute parsed commands
-
-_note: the container always exits 0_
+> Docker container that will manage the flow in a given build
 
 ## Usage
 
 ```bash
-$ docker run screwdrivercd/job-tools:latest ${git_org} ${git_repo} ${git_branch} ${job_name}
+$ docker run -e "SD_TOKEN=${TOKEN}" --rm -it screwdrivercd/job-tools --api-uri=${API_URI} ${BUILD_ID}
```
Comment (d2lam) [inline README.md]: Just wondering, in https://github.com/screwdriver-cd/screwdriver/issues/77 the command is `--api-uri ${API_URI}`, here it's `--api-uri=${API_URI}`

Which one is right?

Comment (stjohnjohnson) [inline README.md]: It should be the same.

**Comment by nkatzman** [comment]
👍

**Pull request merged**